action.yml

```diff
@@ -58,8 +58,10 @@ runs:
     - id: gosec
       run: echo "version=v2.16.0" >> $GITHUB_OUTPUT
       shell: bash
+    # We must remain on 1.9 due to HashiCorp's decision to change the license
+    # of Packer from MPL 2.0 to BSL starting with 1.10.0.
```

Comment:
Did I miss a legal thang that stated that we are unable to use BSL-licensed products for our use case?

Reply:
No. I should probably reword this a bit as we're not strictly prohibited from using BSL-licensed products. However, there are the following considerations:
- The Business Source License is not considered an Open Source license [per SPDX](https://spdx.org/licenses/BUSL-1.1.html).
- There are concerns about compatibility with Terraform 1.6+ with non-HashiCorp tooling for Terraform as a result of the license change.
- We might want to switch to [OpenTofu](https://opentofu.org/) in the future and there is only guaranteed compatibility with Terraform <1.6.
-  The [CNCF](https://www.cncf.io/) [recommends considering alternative components](https://github.com/cncf/foundation/blob/main/source-available-recommendations.md#recommendations) if a project uses a component under the BUSL-1.1 (among others).

Based on the above and team conversations (off GitHub) I was under the impression that we had agreed to remain on versions before the license change. If I misunderstood something I'd like to have a team chat about it but I'm also happy to improve the wording to satisfy any concerns about clarity if the version constraint is amenable.

Reply:
@mcdonnnj - We should definitely flesh out the code comment with the content of [your comment](https://github.com/cisagov/setup-env-github-action/pull/71#discussion_r1440094473).

Reply:
I know we discussed this and I think we are all on the same page, but I still think that we should come to an "official" team consensus and plan our path forward at our next team meeting.  I agree with you both about adding more detail to the code comments in this PR. 👍

Reply:
Please let me know if https://github.com/cisagov/setup-env-github-action/pull/71/commits/afef354747f6e6690f1feeef2558490eaf569005 works for both of you.

Reply:
That looks solid to me, thanks!

@jsf9k please resolve this conversation if you agree.